impls/msft-eshoppen.yaml

```diff
@@ -0,0 +1,96 @@
+name: msft-eshoppen
+description: https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/msft-eShoppen.md
+tags:
+  kind: web
+  complexity: moderate
+  category: cloud
+config:
+  pipeline:
+    calculation: msft-eshoppen
+    normalization: default
+    aggregation: default
+    enrichment: default
+graph:
+  front-end:
+    config:
+      embodied_c: 350 # kgCO2eq
+      reserved_time: 1 # hour
+      resources_reserved: 1
+      total_resources_available: 1
+      lifespan: 4 #years
+    children:
+      cpu:
+        observations:
+          common:
+            server:  Intel® Xeon® Platinum 8272CL
+            tdp: 28 # W
+            carbon_intensity: 951
```

Comment:
Starting to think that grid emissions is just another impact metric and should be treated the same as energy or embodied. As in there is a model that has observations input into it and outputs grid emissions.

If we are going to use the same grid emissions for all calculations then we can just configure a static model on the root node that returns the same value no matter what!